### PR TITLE
Wording: appchains -> networks

### PR DESCRIPTION
**Pull request by augustocollado**
### Description

This PR changes appchains term to networks. 
It also corrects some texts found nearby the replacement and two section names: 
Learn > Appchains in Tanssi -- Learn > Tanssi-Powered Appchains
Build > Tanssi Network -- Build > Network Essentials

This PR doesn't change any url, so no redirects needed.

Note: This PR should be merged **after** the pending content PRs are merged (and the resulting conflicts  solved)

### Checklist

- [ ] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `tanssi-mkdocs` to update redirects

### CODE REVIEW

File: .CONTRIBUTING.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ Below is an example of an `index.md` file:
 
 ```markdown
 ---
-title: Build your Appchain
-description: Install the software to compile yousr appchain, configure your genesis state and core functionalities, test locally and get ready to deploy through Tanssi.
+title: Build your decentralized network
```
Comment (albertov19) [inline .CONTRIBUTING.md:62]:
```suggestion
title: Build your Decentralized Cetwork
```

File: .snippets/text/node-operators/block-producers/onboarding/run-a-block-producer/hardware-requirements.md [hunk 2]
```diff
@@ -15,11 +15,11 @@ The following are some hardware recommendations that have performed well:
...
-|        Network        |    Port     |
-|:---------------------:|:-----------:|
-|   **Tanssi Chain**    | 30333 (TCP) |
-|    **Relay Chain**    | 30334 (TCP) |
-| **Assigned Appchain** | 30335 (TCP) |
+|       Network        |    Port     |
+|:--------------------:|:-----------:|
+|   **Tanssi Chain**   | 30333 (TCP) |
+|   **Relay Chain**    | 30334 (TCP) |
+| **Assigned network** | 30335 (TCP) |
```
Comment (albertov19) [inline .snippets/text/node-operators/block-producers/onboarding/run-a-block-producer/hardware-requirements.md:24]:
```suggestion
| **Assigned Network** | 30335 (TCP) |
```

File: builders/deploy/dapp.md [hunk 3]
```diff
@@ -143,7 +143,7 @@ Finally, [generate and edit](/builders/build/customize/customizing-chain-specs/#
 - `para_id` - within this custom flow, a pre-registered parachain id is required
 - `is_ethereum` - to `true` if exposing Ethereum compatible RPC endpoints is needed
 
-And, depending on whether you are deploying a Snap appchain or a dedicated one, also adjust these attributes:
+And, depending on whether you are deploying a Snap network or a dedicated one, also adjust these attributes:
```
Comment (albertov19) [inline builders/deploy/dapp.md:146]: Will this still be te terminology

File: builders/toolkit/ethereum-api/wallets/subwallet.md [hunk 4]
```diff
@@ -45,30 +45,30 @@ On the following screen, select the method by which you would like to import the
...
 
-1. Paste in the HTTPS RPC URL of your Tanssi appchain. The demo EVM appchain's RPC URL is `https://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network/`. Other parameters will be auto-populated
-2. Paste in the block explorer URL of your Tanssi appchain. The demo EVM appchain's block explorer URL is `https://fra-dancebox-3001-bs.a.dancebox.tanssi.network/`
+1. Paste in the HTTPS RPC URL of your Tanssi network. The demo EVM network's RPC URL is `https://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network/`. Other parameters will be auto-populated
+2. Paste in the block explorer URL of your Tanssi network. The demo EVM network's block explorer URL is `https://fra-dancebox-3001-bs.a.dancebox.tanssi.network/`
 3. Press **Save**
 
-![Add your Tanssi Appchain Network Details in SubWallet](/images/builders/toolkit/ethereum-api/wallets/subwallet/subwallet-6.webp)
+![Add your Tanssi Network Network Details in SubWallet](/images/builders/toolkit/ethereum-api/wallets/subwallet/subwallet-6.webp)
```
Comment (albertov19) [inline builders/toolkit/ethereum-api/wallets/subwallet.md:56]:
```suggestion
![Add your Tanssi Network Network Details in SubWallet](/images/builders/toolkit/ethereum-api/wallets/subwallet/subwallet-6.webp)
```
```suggestion
![Add your Tanssi Network Details in SubWallet](/images/builders/toolkit/ethereum-api/wallets/subwallet/subwallet-6.webp)
```

Comment (albertov19) [reply builders/toolkit/ethereum-api/wallets/subwallet.md:56]: Do a search for "Network Network "to see if you find any other

File: node-operators/block-producers/operational-tasks/upgrade-your-node/index.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Upgrade Block Producers
-description: Follow these instructions to update your node to the latest version of the Tanssi client software you use to produce blocks on the Tanssi Appchain protocol.
+description: Follow these instructions to update your node to the latest version of the Tanssi client software you use to produce blocks on the Tanssi Network protocol.
```
Comment (albertov19) [inline node-operators/block-producers/operational-tasks/upgrade-your-node/index.md:3]:
```suggestion
description: Follow these instructions to update your node to the latest version of the Tanssi client software you use to produce blocks on the Tanssi network protocol.
```

Comment (albertov19) [reply node-operators/block-producers/operational-tasks/upgrade-your-node/index.md:3]: there are some inconsistencies with Network capitalized. search for it and check it is OK

**Comment by augustocollado** [comment]
Updated the URLs, updated the images and added redirects on another PR in the mkdocs repo

**Review by albertov19 — DISMISSED** [review]

**Comment by albertov19** [comment]
As mentioned I think with this new reorg the "Included Templates" can be moved to the Tanssi-Powered Networks section

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**